packages/amplify-graphql-api-construct-tests/src/__tests__/backends/sql-iam-access/app.ts

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env node
```

Comment:
This file is result of splitting the `sql-models.test.ts`.

This file was duplicated from `packages/amplify-graphql-api-construct-tests/src/__tests__/backends/sql-models/app.ts` and then unused pieces were removed.